README.md

```diff
@@ -316,6 +316,36 @@ This cmdlet wraps the functionality of `wsl.exe --shutdown`.
 There is no benefit to using this over `wsl.exe --shutdown`. It is provided purely for the sake of
 completionism.
 
+### Get-WslDistributionOnline
+
+The [`Get-WslDistributionOnline`][] cmdlet gets information about WSL distributions available from online sources.

```

Comment:
Please use a maximum line width of 100 characters in Markdown files (with some leeway for long links).

Reply:
Updated.